eng/helix/content/installnode.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash

# Cause the script to fail if any subcommand fails
set -e
```

 Without this, failures in the pre-command will go unnoticed. https://github.com/dotnet/arcade/issues/2731